### PR TITLE
Feature/forms

### PR DESCRIPTION
**Pull request by smcommits**
# Description

In this project, I have built HTML forms in three different ways; using bare HTML, using the`form_tag` from Ruby and Rails, and finally using the `form_for` tag provided by Ruby on Rails

## Features
The app has the following features
- Users can be created
- Users details can be read
- Users details can be updated

- Validation checks for the following:
    - username is present
    - email is present
    - password is present

## RESTful routes
The app contains the following RESTful routes for the resource `user`:
- `new`
- `create`
- `update`
- `show`

### CODE REVIEW

**Review by javitocor — CHANGES_REQUESTED** [review]
# STATUS CHANGES REQUIRED ♻️ 

Hello! ✋ 
A great job working on this project! 👍 
Very professional readme file 👏 
However, we still need to implement some changes in order to get this project approved 💪 
Kindly check my comments below 👇  apply the change and resubmit your project for a new code review ⏩ 
Good luck! 😺 
## Mandatory changes7
README FILE
- In step 2 in the getting started section it says 'cd micro_reddit' when it should be 'cd re-former', kindly modify it
- Add at least your Linkedin profile in the authors section (Twitter, StackOverflow or any other media if available would be good)
- Add a license
- Add an acknowledgements section, we always should be thankful
CODE
- In the users edit view, kindly remove all the commented out code
- In the HTML form, you should use form_authenticity_token, kindly add it. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_form1.png) [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_form2.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/forms))
- Follow the requirements in your form_tag, it is not working. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_form_tag2.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/forms))

**Review by PhillipUg — APPROVED** [review]
# Status: Project Approved! 🥇 
Congratulations on implementing all project requirements and getting this project approved. :+1:

You can merge this branch into the main branch when you're ready and go on to your next project.
:bulb:  Do not forget to submit your project completion form after merging this branch.

All the best team,

For any assistance, you can contact me on slack as **Phillip Musiime**

**Pull request merged**
